feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go

```diff
@@ -461,36 +515,35 @@ func TestEstablish(t *testing.T) {
 	t.Logf("Start DUT interface Config")
 	configureDUT(t, dut)
 
-	t.Log("Configure Network Instance")
-
-	dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
-	dutConfNIPath.RouterId().Replace(t, dutDst.IPv4)
-
 	// Configure BGP+Neighbors on the DUT
 	t.Logf("Start DUT BGP Config")
-	dutConfPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
+	dutConfPath := dut.Config().NetworkInstance("default").Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
```

Comment:
Please restore the DUT config changes. In particular, "DUT BGP Config before" may fatal because devices had no initial BGP config, and the default network instance should not be replaced at the instance level because it may break other things.

Reply:
Brought in the changes from the ate test